### PR TITLE
support pipeline dml protocol in cdc

### PR DESCRIPTION
**Pull request by hicqu**
This PR adds a `generation` field to `cdc.Event.Row`.

### CODE REVIEW

File: proto/cdcpb.proto [hunk 1]
```diff
@@ -75,12 +75,16 @@ message Event {
         uint64 commit_ts = 2;
         LogType type = 3;
 
+	// generation is for pipelined DML protocol. See kvrpcpb.FlushRequest.generation.
+	uint64 generation = 10;
```
Comment (YuJuncen) [inline proto/cdcpb.proto:79]: Maybe fix the indent?

**Review by YuJuncen — APPROVED** [review]

File: proto/cdcpb.proto (hunk 1 above)
Comment (hicqu) [reply proto/cdcpb.proto:79]: Indent is correct. The line is just wrapped.

**Review by flowbehappy — APPROVED** [review]

File: proto/cdcpb.proto [hunk 2]
```diff
@@ -75,12 +75,16 @@ message Event {
         uint64 commit_ts = 2;
         LogType type = 3;
 
+        // generation is for pipelined DML protocol. See kvrpcpb.FlushRequest.generation.
+        uint64 generation = 10;
+
```
Comment (wuhuizuo) [inline proto/cdcpb.proto:80]: should we put it after the 9th field?

Comment (hicqu) [reply proto/cdcpb.proto:80]: Not necessary. If other guys add more fileds with same id, proto compiler will report it. So we can always group fields by meanning.

**Pull request merged**